### PR TITLE
i18n(zh-cn): Create sitecore.mdx

### PR DESCRIPTION
**Pull request by huyikai**
#### Description (required)

Create sitecore.mdx

#### Related issues & labels (optional)

- #6164

### CODE REVIEW

File: src/content/docs/zh-cn/guides/cms/sitecore.mdx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+1. 根据 Sitcore 的官方文档[创建一个 Sitecore 无头网站](https://doc.sitecore.com/xp/en/developers/sxa/103/sitecore-experience-accelerator/create-a-headless-tenant-and-site.html)。
+2. 在你的终端运行以下项目初始化命令： 
+  ```shell
+  npx @astro-sitecore-jss/create-astro-sitecore-jss@latest
+  ```
+3. 按照终端中的指示创建你的项目。
+
+## 社区资源
+
+- [Astro 的 Sitecore JavaScript 软件开发套件](https://github.com/exdst/jss-astro-public) 在 GitHub 上
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/sitecore.mdx:23]: 语序调整一下
```suggestion
- GitHub 上 [适用于 Astro 的 Sitecore JavaScript 软件开发套件](https://github.com/exdst/jss-astro-public)
```

File: src/content/docs/zh-cn/guides/cms/sitecore.mdx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+- [Astro 中的 Sitecore 简介](https://exdst.com/posts/20231002-sitecore-astro)
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/sitecore.mdx:24]:
```suggestion
- [Sitecore 与 Astro 简介](https://exdst.com/posts/20231002-sitecore-astro)
```

**Review by liruifengv — CHANGES_REQUESTED** [review]

**Review by liruifengv — APPROVED** [review]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks for the great work 🙌

**Pull request merged**